todo_react/todo-react/src/context/context.tsx

```diff
@@ -0,0 +1,46 @@
+import React, {ChangeEvent, useContext} from "react";
+import {ITodo} from "../models";
+
+interface TodosContextModel {
```

Comment:
you don't need ALL this data in context 😅
why `currentTitle` or `onEditChange` are available across whole app?
you should store in context only **common** data, which is needed in multiple components (todos list and methods to change them)